panacea/query_client.go

```diff
@@ -316,3 +318,12 @@ func (q QueryClient) GetOracleRegistration(oracleAddr, uniqueID string) (*oracle
 func (q QueryClient) GetLightBlock(height int64) (*tmtypes.LightBlock, error) {
 	return q.lightClient.TrustedLightBlock(height)
 }
+
+func (q QueryClient) GetOracleParamsPublicKey() ([]byte, error) {
```

Comment:
Why don't we return the parsed oracle public key instead of byte array?
But it's not important for now. I just wonder your thoughts.

Reply:
It doesn't have a big reason. But I think that put the parsing the byte of oracle public key to this function would be more better. So I can remove error handling for parsing public key in `get_oracle_key.go`.

Reply:
Fixed in [a0fe48a](https://github.com/medibloc/panacea-doracle/pull/54/commits/a0fe48a929c83780c86a7930b28e5a5e62dc6885).